src/main/java/org/cascadebot/cascadebot/commands/informational/RolesCommand.java

```diff
@@ -20,10 +20,16 @@
 
     @Override
     public void onCommand(Member sender, CommandContext context) {
-        Table.TableBuilder builder = new Table.TableBuilder("Role ID", "Role Name");
-
+        Table.TableBuilder builder = new Table.TableBuilder("Role ID", "Name", "No. Users", "Colour");
+        
         for (Role role : context.getGuild().getRoles()) {
-            builder.addRow(role.getId(), role.getName());
+            if (role.getName().equals("@everyone")) continue;
+            builder.addRow(
+                    role.getId(),
+                    role.getName(),
+                    String.valueOf(context.getGuild().getMembers().stream().filter(member -> member.getRoles().contains(role)).count()),
+                    role.getColor() == null ?  "Default" : "#" + Integer.toHexString(role.getColor().getRGB())
```

Comment:
The default is transparent really.

Reply:
What would you suggest we put?

Reply:
Instead of default just say transparent. also we should make sure the roles are ordered correctly.

Reply:
+1 for the role ordering
Not sure about saying transparent as it's still an opaque colour

Reply:
Not really as if theirs a color bellow it it goes to that color, and if their isn't it defaults to white/black (depending on theme)

Reply:
Although discord calls it default, so idk

Reply:
@mrjvs Input pls

Reply:
Leave at “default”. We’re working on discord. Stick to the platform.

Reply:
Well yes i do see the point in sticking to the platform, we're building a bot to improve the platform and labeling it as transparent is more correct in the terms of color.

Reply:
I think it's inaccurate to say that it's a transparent colour as it's literally a white colour

Reply:
Only when their isn;t a color bellow it, also it's grey on light theme

Reply:
Can we just leave it on Default? Saves me effort and Lord Jvs said so

Reply:
**Decision from meeting**
We're gonna leave it as `Default`